app/views/casa_admins/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= errors_for(@admin) %>
+
+<h1>Edit Admin</h1>
+
+<div class="row">
+  <div class="col-sm-6">
+    <%= form_for @admin, as: :casa_admin, url: casa_admin_path do |form| %>
+
+    <div class="field form-group">
+      <%= form.label :email %>
+      <%= form.text_field :email, class: "form-control" %>
+    </div>
+
+    <div class="field form-group">
+      <%= form.label :display_name %>
+      <%= form.text_field :display_name, class: "form-control" %>
+    </div>
+
+    <div class="actions">
+      <%= form.submit "Update", class: "btn btn-primary" %>
+      <%= link_to "Back", root_path, class: "btn btn-primary" %>
+    </div>
```

Comment:
Should we wrap this up in a partial for `_form.html.erb` and share the logic with `new.html.erb` and `edit.html.erb`?

Reply:
yes, I'm working on that now 👍🏼